tools/flang1/flang1exe/semfin.c

```diff
@@ -467,7 +467,7 @@ semfin(void)
             error(45, 3, gbl.lineno, SYMNAME(sptr), SYMNAME(gbl.entries));
         pointer_check:
           STYPEP(FVALG(sptr), ST_VAR);
-          if (POINTERG(sptr) || ALLOCATTRG(FVALG(sptr))) {
+          if (POINTERG(FVALG(sptr)) || ALLOCATTRG(FVALG(sptr))) {
```

Comment:
Does the comment below also need an update?

Reply:
Updated. Thanks for all the feedback.